package.json

```diff
@@ -27,12 +27,11 @@
   "bugs": {
     "url": "http://github.com/Azure/azure-sdk-for-js/issues"
   },
+  "scripts": {
+    "prepare": "lerna bootstrap --hoist --ignore-scripts",
+    "build": "lerna run build"
+  },
   "devDependencies": {
-    "@types/node": "^10.10.1",
-    "gulp": "^3.9.1",
-    "ts-node": "^7.0.1",
-    "typescript": "^3.0.3",
-    "webpack": "^4.19.0",
-    "yargs": "^12.0.2"
+    "lerna": "^2.11.0"
```

Comment:
I'm fine with you adding lerna if that makes you more productive, but we have some goals to improve our overall automation that we're going to be using gulp for, so please put the devDependencies back.

Reply:
Oops, definitely.